iocore/net/UnixNet.cc

```diff
@@ -70,7 +70,10 @@ class InactivityCop : public Continuation
       }
 
       // set a default inactivity timeout if one is not set
-      if (ne->next_inactivity_timeout_at == 0 && nh.config.default_inactivity_timeout > 0) {
+      // The event `EVENT_INACTIVITY_TIMEOUT` only be triggered if a read
+      // or write I/O operation was set by `do_io_read()` or `do_io_write()`.
+      if (ne->next_inactivity_timeout_at == 0 && nh.config.default_inactivity_timeout > 0 &&
```

Comment:
I am seeing cases where default_inactivity_timeout is being set on server sessions when ne->read.enabled and ne->write.enabled are 0.  I haven't tracked down if that is required or not for closing the server sessions.

Reply:
@bryancall Curious, do you configure active timeout for Server sessions? If there's no active timeout (which is how we configure for Server sessions), I'd think we should still set the default inactivity timeout just in case.